packages/react-server/src/ReactFlightCache.js

```diff
@@ -9,9 +9,22 @@
 
 import type {CacheDispatcher} from 'react-reconciler/src/ReactInternalTypes';
 
+function createSignal(): AbortSignal {
+  return new AbortController().signal;
+}
+
 export const DefaultCacheDispatcher: CacheDispatcher = {
   getCacheSignal(): AbortSignal {
-    throw new Error('Not implemented.');
+    if (!currentCache) {
```

Comment:
Why unroll this? Does this really help?

Reply:
Just that I don't want to call the Dispatcher and create a dependency on the object wrapper. But this is going away soon anyway as we need a proper lifetime for it.